packages/react-server/src/forks/ReactServerStreamConfig.dom-fb-experimental.js

```diff
@@ -43,7 +43,7 @@ export interface Destination {
 }
 
 export function scheduleWork(callback: () => void) {
-  callback();
+  setTimeout(callback, 0);
```

Comment:
@alunyov I don't know enough about internal FB infra to know whether setImmediate is available all the time or ever. setTimeout is probably not the best choice here but I don't know if we should feature detect setImmediate, use postMessage, or just use setTimeout. Lmk what makes sense for Meta

Reply:
Currently there is none I believe. There’s no async I/O that can be scheduled from JS. Only called into.

So I believe resolving a Promise is the only available for now.

Reply:
updated